src/routes/console/project-[project]/auth/user-[user]/updateLabels.svelte

```diff
@@ -86,12 +86,8 @@
                 <InputTags
                     id="user-labels"
                     label="Labels"
-                    placeholder="Select or tyype user labels"
+                    placeholder="Select or type user labels"
                     bind:tags={labels} />
-                <li>
-                    <Helper type={error ? 'warning' : 'neutral'}
-                        >{error ? error : 'Only alphanumeric characters are allowed'}</Helper>
-                </li>
                 <li class="u-flex u-gap-12 u-margin-block-start-8">
                     {#each suggestedLabels as suggestedLabel}
                         <Pill
```

Comment:
We should be using the interactive tag component here to suggest the labels, not buttons.